pyServer/manifests/linux/hdinsight

```diff
@@ -19,4 +19,10 @@ copy,/var/log/ambari-server/ambari-audit.log
 copy,/var/log/ambari-agent/ambari-agent.log
 copy,/var/log/hadoop-yarn/yarn/*.log
 copy,/var/log/hive/hiveserver2.log
-copy,/var/log/hive/hivemetastore.log
\ No newline at end of file
+copy,/var/log/hive/hivemetastore.log
+copy,/var/log/hdinsight-agent/hdinsight-agent.log
+copy,/var/log/hdinsight-agent/hdinsight-agent.out
+copy,/var/log/hdinsight-startupagent/hdinsight-startupagent.log
+copy,/var/log/hdinsight-startupagent/hdinsight-startupagent.out
+copy,/var/log/hdinsight-provisioning-agent/hdinsight-provisioning-agent.log
+copy,/var/log/hive/hivemetastore.log
```

Comment:
It seems that there is a dup for this file: copy,/var/log/hive/hivemetastore.log